data/distro.yml

```diff
@@ -132,7 +133,7 @@
       <li>
         <h2>Restart</h2>
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
-        <p>Note: Flatpak applications can only be installed using the graphical Software application on either Tumbleweed or Leap 15.0 and later.</p>
+        <p>Note: Flatpak applications can only be installed using the graphical Software application on either openSUSE Tumbleweed or openSUSE Leap 15.0 and later.</p>
```

Comment:
It probably makes sense to drop this notice entirely, since the warning says that everything works fine by default in all supported versions of openSUSE.

Reply:
True. Will fix.

Reply:
If Flatpak is already installed, it would be better to remove the installation instructions altogether. That's what the pages for other distros do.

Reply:
I don't think it is, however. Certainly not on all installation options (i.e. KDE, GNOME, minimal server, etc.).
The note was about that however, apparently on some openSUSE version you had to use GNOME Software rather than the flatpak command line tool..? Tbh, I never experienced that even on older versions of openSUSE.

Reply:
Is GNOME Software installed by default on the GNOME variant of openSUSE? If so, does the 1-click installer of _flatpak_ package also activate its support in GNOME Software so Flatpak applications can be installed using the GNOME Software GUI.

Reply:
GNOME Software is [included in the GNOME pattern](https://build.opensuse.org/package/view_file/openSUSE:Leap:15.1/patterns-gnome/patterns-gnome.spec?expand=1), so yes, Software will be installed by default with the GNOME variant of openSUSE. However, a large part of the openSUSE population uses their OS with KDE.

The `gnome-software` package includes `libgs_plugin_flatpak.so` and requires the `libflatpak` package, so Flatpak support in GNOME Software is a given.
The `flatpak` command-line tool needs to be installed manually.

The "1-Click" installer for `flatpak` has nothing to do with GNOME Software. If you want to install GNOME software using the "1-Click" thing, you need a separate ymp file.

[Also, note that I am quoting "1-Click" for a reason -- it's more like a ten-click installer.]

Reply:
So, what about KDE Discover? Is it installed by default with the Flatpak support included on the KDE variant of openSUSE?